pages/examples/index.tsx

```diff
@@ -111,7 +113,7 @@ export default function Examples({
 
                 <Button
                   as="a"
-                  href={`https://download.biolab.si/download/files/workflows/orange/${download}`}
+                  href={...download}
```

Comment:
@markotoplak 
Although this works. You can't really spread a string. 

`href={download}` would do

Reply:
Thanks. I had no idea what this does, below I saw {...image} somewhere and the usage seemed analogous.